### ISSUE TITLE
SEOTab does not work right as a package dependecy

### ISSUE THREAD

**Issue by DESIGNfromWITHIN** [Sterc/SEOTab#55]
I want to add this in my package, but this does not work, the download button does nothing in the dependencies tab.

```
$attr = array(
    'license' => file_get_contents($sources['docs'] . 'license.txt'),
    'readme' => file_get_contents($sources['docs'] . 'readme.txt'),
    'changelog' => file_get_contents($sources['docs'] . 'changelog.txt'),
    'requires' => array(
        'ace' => '>=1.6.4',
        'formit' => '>=2.2.8',
        'migx' => '>=2.9.6',
        'pdotools' => '>=2.1.20',
        'pthumb' => '>=2.3.3',
        'xrouting' => '>=1.4.1',
        'stercseo' => '>=1.2.2',
    ),
);
```

This does not work either.

```
'SEO Tab' => '>=1.2.2',
```

This also does not work:

```
'seotab' => '>=1.2.2',
```

Is the name SEOtab? stercseo or something else?
What sould I add to add SEOtab as a dependecy?

**Comment by joeke** [comment]
Hi Menno,
It should work when you use 'stercseo' as key in the array. 
Have you tried with an earlier version? Maybe if you use '>=1.2.1'?

**Comment by DESIGNfromWITHIN** [comment]
@joeke Hi I tried it, but 1.2.1 and 1.2.0 don't work...

**Comment by joeke** [comment]
Hi Menno,
I've dug a little deeper into this, and first of all the package name in your requires array should be 'SEO Tab' (and not 'stercseo' as I mentioned before, sorry for that). It seems that the underlying issue is the space in the package name.. I've tested it with another package with spaces in the package name (Google maps TV) and it causes the same issue. I've also added another dependency package without space in package name (FileSluggy) and then there was no issue with downloading the dependency, so I'm pretty sure the space in the name is the issue.
When you manually download the SEO Tab package and install it, the dependencies tab does correctly confirm that the package is installed, so it's only a download issue.
Gonna discuss it with our team tomorrow to determine if it's something we should fix in our packages (removing the spaces from package names) or that it's something that should be fixed in MODX core.
I'll keep you posted..
